src/www/services_unbound_advanced.php

```diff
@@ -350,7 +354,24 @@
                         </td>
                       </tr>
                       <tr>
-                        <td></td>
+                          <td><a id="help_for_extended_statistics" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?= gettext('Extended statistics') ?></td>
+                          <td>
+                              <input name="extended_statistics" type="checkbox" id="extended_statistics" value="yes" <?= empty($pconfig['extended_statistics']) ? '' : 'checked="checked"' ?> />
+                              <div class="hidden" data-for="help_for_extended_statistics">
+                                  <?= gettext("If  enabled, extended  statistics are printed. Default is off, because keeping track of more statistics takes time.") ?>
```

Comment:
```suggestion
                                  <?= gettext("If enabled, extended statistics are printed.") ?>
```

although the former matches the documentation's wording it is a bit redundant :)